packages/web/src/components.d.ts

```diff
@@ -9,10 +9,12 @@ import { Validator, ValidatorEntry } from "./validators";
 import { SpacingValues } from "./utils/types/spacing";
 import { ContentValues, GridGapValues } from "./components/gcds-grid/gcds-grid";
 import { RadioObject } from "./components/gcds-radio-group/gcds-radio-group";
+import { RadioObject as RadioObject1 } from "./components/gcds-radios/radio";
 export { Validator, ValidatorEntry } from "./validators";
 export { SpacingValues } from "./utils/types/spacing";
 export { ContentValues, GridGapValues } from "./components/gcds-grid/gcds-grid";
 export { RadioObject } from "./components/gcds-radio-group/gcds-radio-group";
+export { RadioObject as RadioObject1 } from "./components/gcds-radios/radio";
```

Comment:
I wonder why it's duplicated?

Reply:
Both components use a `RadioObject` that is exported. This won't happen after we remove `gcds-radio-group`

Reply:
I think my eyes were tired and thought I was seeing two `imports`, but they seem to be `import`, and then `export`!